### PR TITLE
Vid

### PR DESCRIPTION
**Pull request by FatMouse-productions**
did some cool shit

### CODE REVIEW

**Comment by Aritton** [comment]
This shit is okay.

File: huhu.txt [hunk 1]
```diff
@@ -1 +1,5 @@
-huehue
\ No newline at end of file
+huehue
+
+consider this: hjoh hjoh hjoh
+
```
Comment (Aritton) [inline huhu.txt:4]: delete everything below!

**Review by Aritton — CHANGES_REQUESTED** [review]
Not good. change the file.

File: huhu.txt (hunk 1 above)
Comment (FatMouse-productions) [reply huhu.txt:4]: no i wont

Comment (Aritton) [reply huhu.txt:4]: Ok.

**Review by Aritton — APPROVED** [review]
Everything is okay!

**Pull request merged**